master/usage/troubleshooting/faq.md

```diff
@@ -266,16 +266,34 @@ you can configure incoming NAT with port mapping on the host on which the contai
 is running on.
 
 ```
-iptables -A PREROUTING -t nat -i eth0 -p tcp --dport <EXPOSED_PORT> -j DNAT  --to <CALICO_IP>:<SERVICE_PORT>
```

Comment:
Probably shouldn't block this PR, but we should be directing people to the right orchestrator documentation for this sort of stuff as well.  

For most users this isn't how incoming traffic is supposed to work in Kubernetes, for example, and we shouldn't be recommending this approach to Kubernetes users (unless they have special requirements)